includes/definitions/gigabyte-bmc.yaml

```diff
@@ -0,0 +1,15 @@
+os: gigabyte-bmc
+type: server
+group: unix
+mib_dir: gigabyte-bmc
+text: 'GIGABYTE-BMC'
+over:
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+    - { graph: device_storage, text: 'Storage Usage' }
+discovery:
+    -
+      snmpget:
+          oid: .1.3.6.1.4.1.15370.550.2.0 #minPasswordLength, value set by GIGABYTE, cannot be changed.
```

Comment:
Is there really nothing more unique to match on?

Reply:
Unfortunately no. I can send you a whole SNMPbulkwalk result. there is no sysDescr, and sysObjectID is "generic".

Reply:
There should be another item or two here so the snmpget is not done for every device.